pkgs/applications/networking/irc/epic5/default.nix

```diff
@@ -17,14 +17,14 @@ stdenv.mkDerivation rec {
 
   postConfigure = ''
     substituteInPlace bsdinstall \
-      --replace /bin/cp cp \
-      --replace /bin/rm rm \
-      --replace /bin/chmod chmod
+      --replace /bin/cp ${pkgs.coreutils}/bin/cp \
```

Comment:
You should be able to use `coreutils` as an argument directly instead of going through the package set.